src/hooks/useForm.jsx

```diff
@@ -115,6 +115,27 @@ const useForm = ({ validateOnChange = false } = {}) => {
 		return [name, values];
 	};
 
+	/**
+	 * @function
+	 * @name getRadioButtonGroupValue
+	 * @description Returns the checked value inside the group in a format that can be used with Object.fromEntries().
+	 *
+	 * @author Timothée Simon-Franza
+	 *
+	 * @param {object} radioButtonGroup The radio button group reference to retrieve the value from.
+	 *
+	 * @return {array}
+	 */
+	const getRadioButtonGroupValue = (radioButtonGroup) => {
+		const { isRadioButtonGroup, name, rules, ...fields } = radioButtonGroup;
```

Comment:
Aren't `isRadioButtonGroup` and `rules` unused here ?

Reply:
Yes it's because we need to ignore them to retrieve only our fields, which are registered dynamically using their value.